cli/tests/integration/fmt_tests.rs

```diff
@@ -27,49 +29,37 @@ fn fmt_test() {
   let badly_formatted_json_str = badly_formatted_json.to_str().unwrap();
   std::fs::copy(badly_formatted_original_json, &badly_formatted_json).unwrap();
   // First, check formatting by ignoring the badly formatted file.
-  let status = util::deno_cmd()
-    .current_dir(&testdata_fmt_dir)
-    .arg("fmt")
-    .arg(format!(
-      "--ignore={badly_formatted_js_str},{badly_formatted_md_str},{badly_formatted_json_str}"
-    ))
-    .arg("--check")
-    .arg(badly_formatted_js_str)
-    .arg(badly_formatted_md_str)
-    .arg(badly_formatted_json_str)
-    .spawn()
-    .unwrap()
-    .wait()
-    .unwrap();
+
+  let output = context
+    .new_command()
+    .cwd("fmt")
+    .args(format!("fmt --ignore={badly_formatted_js_str},{badly_formatted_md_str},{badly_formatted_json_str} --check {badly_formatted_js_str} {badly_formatted_md_str} {badly_formatted_json_str}"))
+    .run();
+
   // No target files found
-  assert!(!status.success());
+  output.assert_exit_code(1);
+  output.skip_output_check();
 
   // Check without ignore.
-  let status = util::deno_cmd()
-    .current_dir(&testdata_fmt_dir)
-    .arg("fmt")
-    .arg("--check")
-    .arg(badly_formatted_js_str)
-    .arg(badly_formatted_md_str)
-    .arg(badly_formatted_json_str)
-    .spawn()
-    .unwrap()
-    .wait()
-    .unwrap();
-  assert!(!status.success());
+  let output = context
+    .new_command()
+    .cwd("fmt")
+    .args(format!("fmt --check {badly_formatted_js_str} {badly_formatted_md_str} {badly_formatted_json_str}"))
+    .run();
+
+  output.assert_exit_code(1);
+  output.skip_output_check();
 
   // Format the source file.
-  let status = util::deno_cmd()
-    .current_dir(&testdata_fmt_dir)
-    .arg("fmt")
-    .arg(badly_formatted_js_str)
-    .arg(badly_formatted_md_str)
-    .arg(badly_formatted_json_str)
-    .spawn()
-    .unwrap()
-    .wait()
-    .unwrap();
-  assert!(status.success());
+  let output = context
+    .new_command()
+    .cwd("fmt")
+    .args(format!("fmt {badly_formatted_js_str} {badly_formatted_md_str} {badly_formatted_json_str}"))
```

Comment:
And here

Reply:
After updating this one to `args_vec` for some reason the formatted changes to the testdata dir remain after running the tests, unlike on main